# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../firefox/bigendian.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-81.0.2/config/external/icu/data/moz.build.vanilla	2020-10-17 12:31:24.250018267 +0200
+++ firefox-81.0.2/config/external/icu/data/moz.build	2020-10-17 12:31:35.014018209 +0200
@@ -22,7 +22,7 @@
     'little': SRCDIR,
     'big': OBJDIR,
 }
-endianness = CONFIG.get('TARGET_ENDIANNESS', 'little')
+endianness = CONFIG['TARGET_ENDIANNESS']
 DEFINES['ICU_DATA_FILE'] = '"%s/%s"' % (data_dir[endianness], data_file[endianness])
 DEFINES['ICU_DATA_SYMBOL'] = '%sicudt%s_dat' % (prefix, CONFIG['MOZ_ICU_VERSION'])
 SOURCES += [
